ng-client/src/app/app.module.ts

```diff
@@ -1,29 +1,18 @@
 import { BrowserModule } from '@angular/platform-browser';
-import { NgModule, CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
-import { HttpModule } from '@angular/http';
-import { FormsModule } from '@angular/forms';
+import { NgModule } from '@angular/core';
```

Comment:
Yeah, i'm looking at cleaning that up as well. This has to do with how we're including the angular material modules. If we include them correctly, we'll get rid of these errors.